packages/amplify-migration-tests/src/__tests__/migration_tests_v10/api-graphql-v2.migration.test.ts

```diff
@@ -0,0 +1,74 @@
+import { addApiWithoutSchema, 
```

Comment:
Please run prettier on the PR. Indentation looks off and single quotes should be the default choice for string literals.